turbo/snapshotsync/snapshotsync.go

```diff
@@ -185,6 +185,12 @@ Loop:
 				}
 
 				dbg.ReadMemStats(&m)
+				memStats, err := dbg.GetMemUsage()
```

Comment:
we now have a 2 funcs - `dbg.ReadMemStats` and `dbg.GetMemUsage` - would be nice to find a way to hook in your new changes into `ReadMemStats` to avoid confusion "what does first func do and what does the 2nd func do"

Reply:
talked further offline - maybe need better naming like dbg.ReadVirtualMemStats and moving this to a background goroutine which logs periodically similar to GoodPeers